src/index.js

```diff
@@ -0,0 +1,84 @@
+import './style.css';
+import menuImgSrc from './menu.png';
+import enterImgSrc from './enter.png';
+import refreshImgSrc from './refresh.png';
+
+const body = document.getElementsByTagName('body')[0];
+const container = document.querySelector('.to-do-list');
+
+const h1Container = document.querySelector('.h1-container');
+const refreshImg = document.createElement('img');
+refreshImg.src = refreshImgSrc;
+refreshImg.className = 'refresh-img';
+h1Container.appendChild(refreshImg);
+const form = document.createElement('form');
+const input = document.createElement('input');
+input.type = 'text';
+input.name = 'description';
+input.placeholder = 'Add to your list...';
+const inputLine = document.createElement('div');
+inputLine.className = 'input-line';
+inputLine.appendChild(input);
+const enterImg = document.createElement('img');
+enterImg.src = enterImgSrc;
+enterImg.className = 'enter-img';
+
+inputLine.appendChild(enterImg);
+form.appendChild(inputLine);
+container.appendChild(form);
+
+const arryOfTasks = [];
+
+const Task = {
+  create(description, index) {
+    const task = Object.create(this);
+    task.description = description;
+    task.completed = false;
+    task.index = index;
+    return task;
+  },
+};
+
+input.addEventListener('keydown', (e) => {
+  if (e.key === 'Enter' && input.value !== '') {
+    e.preventDefault();
+    const task = Task.create(input.value, arryOfTasks.length);
+    arryOfTasks.push(task);
+    input.value = '';
+    localStorage.setItem('tasks', JSON.stringify(arryOfTasks));
+  }
+});
+
```

Comment:
- [ ] Great work up to this point but there are some crucial aspects you have not met yet.
  - Currently, when I add a task, I have to refresh a page to see it. Kindly follow  this [video](https://www.youtube.com/watch?v=AcUd-_Yjjqg) to guide you on the implementation
  - Kindly note that the property index should be set to the value of the new array length (i.e. if you're adding a 5th task to the list, the index of that task should equal to 5).
  
![image](https://user-images.githubusercontent.com/98453979/235925428-0ce28fc0-724b-4c56-ac03-7ab784f95964.png)

   - I was not able to edit and delete the tasks. Kindly implement these functionalities as they are very crucial to the application